source/CardGames.py

```diff
@@ -203,3 +216,23 @@ def dealCards(self, numCards: int, players: "list[Player]"):
   def resetDeck(self):
     self.deck.reset()
     self.deck.shuffle()
+
+def Play(dealer: Dealer, players: list):
+  players_passed = []
+  while len(players) > 0:
+    for player in players:
+      player.display()
+      draw = input("Enter 'draw' or 'pass': ")
+      if draw == "draw" or draw == "Draw":
+        dealer.dealCards(1, [player])
+        print("Card drawn by %s.\n" % (player.name))
+        player.display()
+        if player.bust() == True:
+          print("You bust!\n")
+          players.pop(players.index(player))
+      else:
+        print("%s passed.\n" % (player.name))
+        players_passed.append(player)
+        players.pop(players.index(player))
+      input("Next Player press 'Enter' when ready!")
+  return players_passed
```

Comment:
This is already in the dev_0 branch remove this before merging

Reply:
This is my feature brah